src/importc.h

```diff
@@ -47,4 +47,4 @@
 /****************************
  * Define it to do what other C compilers do.
  */
-#define __builtin_offset(t,i) ((size_t)((char *)&((t *)0)->i - (char *)0))
+#define __builtin_offsetof(t,i) ((size_t)((char *)&((t *)0)->i - (char *)0))
```

Comment:
Isn't builtin_offsetof also supposed to work with bit-fields?

Reply:
```
C11 7.19.3 (If the specified member is a bit-field, the behavior is undefined.)
```